terrautils/metadata/lemnatec.py

```diff
@@ -1,24 +1,32 @@
-'''
-Create a cleaned/curated metadata object based on the LemnaTec fixed metadata
-Some metadata used for search
-Some metadata used for extractors
+"""
+Given a JSON object containing the original LemnaTec metadata, convert
+field names and values to standardized formats required for TERRA-REF.
```

Comment:
No -- "...and possibly refactor to align with standards such as the OGC..."
Even the primary sensor-metadata repository currently doesn't align with SWE. It still contains mainly the LemnaTec fields.  

The cleanup process is "standardizing" the LemnaTec fields in the sense of removing spaces, fixing misspellings, consolidating multiple of the same field (time, Time, Timestamp, timestamp), and trying to account for units where known.  As far as I know, there is no standard that will help us address "scanner_west_location_in_camera_box_m" (was "scanner west location in camera box [m]") in time for the code freeze.